archapp/mgmt.py

```diff
@@ -47,7 +47,7 @@ def search_pvs(self, glob, do_print=True):
         url += PV_ARG.format(glob)
         # Increase timeout on search calls because they take approx 5-6 secs
         # Call time is mostly independent of which glob we use
-        pvs = get_json(url, timeout=10)
+        pvs = get_json(url, timeout=20)
```

Comment:
Really this should be an argument we pass through to `search_pvs` and other similar methods.  Not on you for this PR, but we can add it to the list of things to do on our next maintenance pass.